static/css/components/mybooks-list.less

```diff
@@ -26,6 +26,14 @@
     }
   }
 
+  h4.work-subtitle {
```

Comment:
would have been great if we could have gotten away w/ not adding additional css! Keep it simple, re-use what we can :)

Reply:
If we plan to restructure the semantic I will remove the additional CSS.